front/src/Factur.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import DatePicker from 'react-datepicker';
+// import moment from 'moment';
+ 
+import 'react-datepicker/dist/react-datepicker.css';
+
+
+export class Factur extends React.Component<any,any> {
+  state = {
+    startDate: new Date(),
+    title: '',
+    name: '',
+    lastName: '',
+    netto: ''
+  };
+
+  handleNameChange = (e:any) => {
+    this.setState({
+      name: e.target.value
+    });
+  }
+  handleTitleChange = (e:any) => {
+    this.setState({
+      title: e.target.value
+    });
+  }
+  handleLastNameChange = (e:any) => {
+    this.setState({
+      lastName: e.target.value
+    });
+  }
+  handleNettoChange = (e:any) => {
+    this.setState({
+      netto: e.target.value
+    });
+  }
+  handleChange = (date:any) => {
+    this.setState({
+      startDate: date,
+    });
+  };
+  render(){
+    const { handleSubmit, submitting } = this.props
+    return (
+        <form onSubmit={handleSubmit} className="form-group col-6 container">
+          <h1>Utwórz fakture</h1>
+          <div>
+            <div>Tytuł</div>
```

Comment:
Czemu tytuł to div a nie label? (to samo tyczy się pozostałych inputów)

Reply:
ok